src/game.cpp

```diff
@@ -3725,7 +3725,8 @@ std::vector<std::string> game::list_active_characters()
  */
 void game::write_memorial_file(std::string sLastWords)
 {
-    const std::string &memorial_dir = FILENAMES["memorialdir"];
+    const std::string &memorial_dir = FILENAMES["memorialdir"] +  world_generator->active_world->world_name + "/";
```

Comment:
Look at path_info.cpp - the `/` has been used all over the place and has not caused any trouble there, so why should it here?